src/plotly_recipes.jl

```diff
@@ -94,7 +100,7 @@ function _dataframe_plots_internal(
     layout_kwargs[:xaxis] =
         Plots.PlotlyJS.attr(; showticklabels = !bar, title = "$time_interval")
     layout_kwargs[:title] = "$title"
-    layout_kwargs[:barmode] = stack ? "stack" : "group"
+    layout_kwargs[:barmode] = stack ? "relative" : "group"
```

Comment:
I don't object to this change, but my test stacked plot with negative data looks the same before vs. after. In what situation does this give a different-looking plot?

Reply:
when generation is negative. That's the use case